### PR TITLE
Registration flow fixes

### PR DESCRIPTION
**Pull request by dhuangg**
fixed a couple of small bugs in the registration cart

### CODE REVIEW

File: src/components/FeatureViews/Search/SearchResults.js [hunk 1]
```diff
@@ -53,16 +49,31 @@ const SearchResults = () => {
...
+		switch (filter) {
+			case "account":
+			case "course": {
+				return 12;
+			}
+			default: {
+				return 4;
+			}
+		}
+	};
```
[the remark this answers is not in the document]
Comment (dhuangg) [reply src/components/FeatureViews/Search/SearchResults.js:63]: default is when the filter is set to "all" in which case the param is "null" hence we need a default case.

File: src/components/FeatureViews/Registration/RegistrationCart.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ const RegistrationCart = () => {
 
   const courseIDs = useMemo(
 	  () => {
-		  if (registered_courses.length > 0) {
+		  if (Object.values(registered_courses).length > 0) {
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Registration/RegistrationCart.js:41]: Unless you specifically want `courseIDs` to be undefined if there are no registered courses, remove this `if` statement and `courseIDs` will simply be an empty array if there are no registered courses -  `Object.values` returns an empty array, and the subsequent operations do not change the empty array.

File: src/components/FeatureViews/Search/SearchResults.js [hunk 3]
```diff
@@ -50,17 +43,10 @@ const SearchResults = () => {
...
-			}
-			default: {
-				return 4;
-			}
-		}
-	};
+  const getPageSize = (filter) => filter ? {
+    "account": 12,
+    "course": 12,
+  } : 4;
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Search/SearchResults.js:49]: lol this wont actually work how you want it to. you need to index the object like `{...}[filter]`

File: src/components/FeatureViews/Search/SearchResults.js [hunk 4]
```diff
@@ -19,19 +19,15 @@ import CourseFilters from "./CourseFilters";
 import CoursesCards from "./cards/CoursesCards";
 import NoResultsPage from "./NoResults/NoResultsPage";
 import SearchResultsLoader from "./SearchResultsLoader";
-
-const toAPIPage = (page) => Math.floor((page + 1) / 2);
+import {capitalizeString} from "../../../utils";
```
[the remark this answers is not in the document]
Comment (dhuangg) [reply src/components/FeatureViews/Search/SearchResults.js:22]: let me be, neil. my ide did this. LOL

**Pull request merged**